### PR TITLE
HistogramViewer support for arch_printing dashboards

### PR DESCRIPTION
**Pull request by slack0**
Files:
- modified:   __init__.py
- new file:   viewer_histogram.py
- modified:   version.py

* Added HistogramViewer to correctly generate histograms based on quad() Glyph renderer in Bokeh
* Incremented version number to 0.4.6
* Updated relative imports

### CODE REVIEW

File: iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        x_sciformat (bool): Whether to display the x tick values in 10^X format
+            or in normal decimal.  Defaults to scientific.
+        y_decade_mantissa (list): The mantissa values to use for the y_axis.  If
+            not specified this defaults to ticks on 1, 2 and 5 mantiss values.
+        y_sciformat (bool): Whether to display the y tick values in 10^X format
+            or in normal decimal.  Defaults to scientific.
+    """
+
+    def __init__(self, **kwargs):
+
```
Comment (timburke) [inline iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py]: This extra space isn't needed.

File: iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    All functionality in BaseViewer is present and there are a few
+    convenience functions for setting the tick format on the axes
+    easily.
+
+    Apart from the special arguments listed below, all other arguments
+    are passed through to BaseViewer.
+
+    Args:
+        left
```
Comment (timburke) [inline iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py]: I think this docstring line got added accidentally.

**Comment by dkarchmer** [comment]
Always a good idea to include a screen shot when submitting a PR for a UI change, so reviewer can visualize the change.

File: iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py (hunk 1 above)
Comment (slack0) [reply iotile_analytics_interactive/iotile_analytics/interactive/viewer_histogram.py]: Got it. Will remove these extra spaces and commit.

**Pull request merged**